package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metrics/metric",
-  "version": "1.0.0",
+  "version": "2.0.0",
```

Comment:
how is this still at 1.0.0? The latest published version is already 2.0.0

Reply:
Bah. I forgot to commit it to git after publishing to npm.

Reply:
ah right of course